variables.tf

```diff
@@ -46,6 +46,32 @@ variable "burstable_mode" {
   default     = null
 }
 
+variable "instance_market_options_enabled" {
+  type        = bool
+  description = "Wheter to enable the purchasing option for the instances"
+  default     = false
+}
+
+variable "market_type" {
+  type        = string
+  description = "Type of market for the instance. Valid values are `spot` and `capacity-block`. Defaults to `spot`. Required if `spot_options` is specified."
```

Comment:
```suggestion
  description = "Type of market for the instance. Valid values are `spot` and `capacity-block`. Defaults to `spot`. Required if a non-empty value is provided for `spot_options_attributes`."
```

Reply:
suggestion committed, thanks for the input @joe-niland